injector/dns_disruption_test.go

```diff
@@ -36,31 +36,32 @@ var _ = Describe("Failure", func() {
 	BeforeEach(func() {
 		// cgroup
 		cgroupManagerMock = mocks.NewCGroupManagerMock(GinkgoT())
-		cgroupManagerMock.EXPECT().RelativePath(mock.Anything).Return("/kubepod.slice/foo")
-		cgroupManagerMock.EXPECT().Write(mock.Anything, mock.Anything, mock.Anything).Return(nil)
+		cgroupManagerMock.EXPECT().RelativePath(mock.Anything).Return("/kubepod.slice/foo").Maybe()
```

Comment:
I am not really a fan of the `Maybe` keyword, because it does not check if the method has been called during the `AssertExpectation` and let the suit test be successful. I think the tests should be reworked in order to define `EXPECT` when it is needed.

Reply:
In a first time we can allow the `Maybe`, but it must be refactored in another PR.

Reply:
Indeed, I won't rework those tests here, the PR is big enough (an `Maybe` is exactly was the initial implementation of those tests was doing, hence correct)

I will create an issue to rework those and the network ones, thanks for pointing that out.

Reply:
done